indico/web/client/styles/partials/_main.scss

```diff
@@ -222,36 +225,117 @@ nav.main-breadcrumb {
 
 /* Styles for the session bar at the top right corner */
 
-div.session-bar {
+#session-bar {
   @include font-family-title();
 
-  margin: 0.5em 0.5em 0 0;
+  flex: none;
+  display: flex;
+  align-items: center;
+  gap: 0.1em;
```

Comment:
Not a big fan of the gap between the buttons to be honest. Could we either remove it, or use some subtle separator?

<img width="343" height="62" alt="Image" src="https://github.com/user-attachments/assets/f0655c5f-102a-4e8d-b31b-1ae5d17130a6" />

<img width="345" height="50" alt="Image" src="https://github.com/user-attachments/assets/912f4558-7608-499e-b290-9a215d95ae4d" />